### DISCUSSION TITLE
'Verification' of IDN domain

### DISCUSSION THREAD

**Discussion by habi** [mastodon/mastodon#20929] [Q&A]
I'm 'blessed' with an Umlaut surname and have an IDN domain name, https://davidhaberthür.ch/
In the HTML code there, I have `<a rel="me" href="https://mastodon.social/@habi">mastodon.social/@habi</a>` since about a good week.

Mastodon.social does not give me a nice green checkmark on the URL which I've added to my profile.
Is that maybe related to the IDN domain name, or how can I get the URL verified in my profile?

**Comment by ineffyble** [comment]
I think the issue is that your profile contains the domain as text, not a URL/link. Try editing it and adding `https://`

**Reply by habi** [reply]
Well, that might turn out to be embarrassing :man_facepalming:
I've seen all those URLs in different profiles but never thought I'd need to add them *with* http(s)://.
I've done that just now and wait for a bit to see if it gets verified.

**Reply by ineffyble** [reply]
Looking at the codebase, I'm not sure it will help.

You do need to have a URL, but also, verification of IDN domains is explicitly prevented for security reasons right now.

**Reply by ineffyble** [reply]
https://github.com/mastodon/mastodon/pull/20295

**Reply by habi** [reply]
Well, that's a bummer! I see the possible implications, but think it would be good to at least accept punycode for such domains...
